src/tests/ftest/soak/utils.py

```diff
@@ -1134,6 +1134,64 @@ def create_fio_cmdline(self, job_spec, pool):
     return commands
 
 
+def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.